### DISCUSSION TITLE
Storyboard on Beatmap Info Wedge/Background

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#22433] [Ideas]
Took this idea from #22426.

Instead of letting the gameplay/storyboard play only on main menu, Why not straight up play it on beatmap selection as well? Maybe on the beatmap info wedge or in the background as well? I think it would look great for osu!supporters.

Plus I think it fits even after the beatmap redesign on Figma.
![image](https://user-images.githubusercontent.com/64581009/214922895-18a3adb6-4e77-4028-a2d9-793013b1296a.png)

**Comment by mk56-spn** [comment]
I don't see how it would fit given the redesign bar is thinner than this one and this one was already too thin for 16:9 or 4:3 . You'd have to drop half the height of the video

**Reply by mk56-spn** [reply]
Something i see working better perhaps is moving the image inside the frame to give it some amount of motion

**Reply by NotGumballer91** [reply]
It doesn't have to show the entire video. It's a preview after all. The main menu with storyboard covers that
